kuwala/core/backend/README.md

```diff
@@ -129,4 +129,39 @@ We are using dbt for doing transformations on top of data warehouses, so we are
 - schema_name
 - table_name
 
-</details>
\ No newline at end of file
+</details>
+
+## Transformations
+
+To add a new transformation, follow these instructions: TBD
+
+The following transformations are currently available:
+
+- General
+  <details>
+    <summary>Group by</summary>
+  
+    Hint: Syntax for aggregated columns is `COLUMN_NAME` + "KUWALA_AGG" + `COLUMN_AGG`.
+  </details>
+  
+  - Is false
+  - Is not null
+  - Is true
+  - Remove duplicates
+  - Replace null
```

Comment:
Group by column is not in the readme but exists in the transformation category item

Reply:
`group_by` is in the README. But I wrapped it in `<details>` because there is a special syntax for the parameters that needs to be used in the frontend/when sending API calls.

![image](https://user-images.githubusercontent.com/28758040/167863306-c53c9717-f5c3-40f9-8dd6-7d81057c7fbb.png)